gtest/src/program.rs

```diff
@@ -172,6 +173,19 @@ impl From<&str> for ProgramIdWrapper {
     }
 }
 
+#[macro_export]
+macro_rules! state_args {
+    () => {
+        Option::<()>::None
```

Comment:
disagree with unit type here. as I think you did it to satisfy ur own : Encode bound, but it's incorrect, go with `Option::<Vec<u8>>::None` instead

Reply:
but `state_args!` constructs `Option<(arg1, arg2, ...)> where arg1: Encode, arg2: Encode, ...` so the `()` is here for consistency